Docker/eosc.sh

```diff
@@ -4,7 +4,7 @@
 # Go into cmd loop: sudo ./eosc.sh
```

Comment:
we should probably rename this and all of its in docker dependencies to `eosioc` to match the binaries

Reply:
I updated the Docker/* with eosio. I will have our docker guy test and fix as needed.